BackofficeBundle/Form/Type/AreaType.php

```diff
@@ -60,15 +64,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'label' => 'open_orchestra_backoffice.form.area.bo_percent'
         ));
         if(!array_key_exists('disabled', $options) || $options['disabled'] === false){
-            $builder->addEventSubscriber(new AreaCollectionSubscriber($this->areaClass));
+            $builder->addEventSubscriber(new AreaCollectionSubscriber($this->areaClass, $this->translator));
```

Comment:
As there are a lot of dependancies for this service, could we declare this subscriber as a service and pass it directly to the `AreaType`